src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/wrapper/DraftOrdersWrapper.java

```diff
@@ -115,6 +117,13 @@ public void appendAgreedDraftOrderCollection(List<AgreedDraftOrderCollection> ne
         agreedDraftOrderCollection.addAll(newAgreedDraftOrderCollection);
     }
 
+    public void appendIntvAgreedDraftOrderCollection(List<AgreedDraftOrderCollection> newAgreedDraftOrderCollection) {
+        if (intvAgreedDraftOrderCollection == null) {
+            intvAgreedDraftOrderCollection = new ArrayList<>();
+        }
+        intvAgreedDraftOrderCollection.addAll(newAgreedDraftOrderCollection);
+    }
+
```

Comment:
Looks OK, but I'm making assumptions about what it's doing.  Could we have some Javadoc?